examples/remove_CR_hits.py

```diff
@@ -0,0 +1,147 @@
+"""
+========================
+Removing Cosmic Ray Hits
+========================
+
+This example illustrates how to remove cosmic ray hits from a LASCO C2 image (both `FITS` and `jp2`)
+using `astroscrappy.detect_cosmics`. Astroscrappy is not a `sunkit_image` subroutine and can be
+installed seperately using `pip` or `conda`.
+"""
+# sphinx_gallery_thumbnail_number = 2
+
+import matplotlib.pyplot as plt
+
+from sunpy.map import Map
+
+import astroscrappy
+
+###############################################################################
+# First, we will work with the FITS files.
+# These imports are necessary to download the FITS file based on the instrument.
+# For more details on how to download a particular LASCO FITS file as a map see
+# Sunpy's example `downloading_lascoC3.py`
+from sunpy.net import Fido, attrs as a
```

Comment:
You want to add a comment that this will require sunpy installed with all of its "net" depencides.